packages/tokens/src/figma-tokens/_docs.json

```
@@ -851,13 +915,125 @@
       }
     },
     "select": {
-      "description": {
-        "value": "Selects allow you to choose items from a menu of predefined options.",
-        "type": "text"
+      "default": {
+        "description": {
+          "value": "Selects allow you to choose items from a menu of predefined options.",
+          "type": "text"
+        },
+        "title": {
+          "value": "Select",
+          "type": "text"
+        }
       },
-      "title": {
-        "value": "Select",
-        "type": "text"
+      "labels": {
+        "title": {
+          "value": "Labels",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the label attribute to give the select an accessible label. For labels that contain HTML, use the label slot instead.",
+          "type": "text"
+        }
+      },
+      "help-text": {
+        "title": {
+          "value": "Help Text",
+          "type": "text"
+        },
+        "description": {
+          "value": "Add descriptive help text to a select with the help-text attribute. For help texts that contain HTML, use the help-text slot instead.",
+          "type": "text"
+        }
+      },
+      "clearable": {
+        "title": {
+          "value": "Clearable",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the clearable attribute to make the control clearable. The clear button only appears when an option is selected.",
+          "type": "text"
+        }
+      },
+      "read-only": {
+        "title": {
+          "value": "Read-only",
+          "type": "text"
+        },
+        "description": {
+          "value": "Add the read-only attribute to draw a read-only select.",
+          "type": "text"
+        }
+      },
+      "disabled": {
+        "title": {
+          "value": "Disabled",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the disabled attribute to disable a select.",
+          "type": "text"
+        }
+      },
+      "multiple": {
+        "title": {
+          "value": "Multiple",
+          "type": "text"
+        },
+        "description": {
+          "value": "To allow multiple options to be selected, use the multiple attribute. It’s a good practice to use clearable when this option is enabled. To set multiple values at once, set value to a space-delimited list of values.",
+          "type": "text"
+        }
+      },
+      "initialvalue": {
+        "title": {
+          "value": "Setting Initial Values",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the value attribute to set the initial selection.\n\nWhen using multiple, the value attribute uses space-delimited values to select more than one option. Because of this, <sl-option> values cannot contain spaces. If you’re accessing the value property through Javascript, it will be an array.",
```

`<sl-option>` should be exchanged with `<syn-option>`

```suggestion
          "value": "Use the value attribute to set the initial selection.\n\nWhen using multiple, the value attribute uses space-delimited values to select more than one option. Because of this, <syn-option> values cannot contain spaces. If you’re accessing the value property through Javascript, it will be an array.",
```